app/assets/javascripts/copo/copo.edit-checkin.es6

```diff
@@ -87,24 +144,29 @@ window.COPO.editCheckin = {
 
   updateCheckin(response) {
     // tries to find the checkin in gon and update it with the response
-    checkin = _.find(gon.checkins, _.matchesProperty('id',response.id));
-    checkin.lat = response.lat;
-    checkin.lng = response.lng;
-    checkin.edited = response.edited;
+    checkin = _.find(gon.checkins, _.matchesProperty('id', response.checkin.id));
+    checkin.lat = response.checkin.lat;
```

Comment:
@t-harps I guess, edited is missing in serializer, you're right.
But it also is not presented [here](https://github.com/earlymarket/Coposition/blob/cf54c13cb30030c1542b450d14a1b41ab3511734/app/models/device.rb#L49)